docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 version: '3'
 services:
+
     web:
-        build: ./
+        build: ./.docker/web
         container_name: svgtranslate_web
         ports:
           - ${TOOLFORGE_DOCKER_PORT}:80
         volumes:
-          - ./.docker/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
+          - ./.docker/web/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
           - ./:/var/www:cached
         environment:
           TOOLFORGE_DOCKER_PORT: ${TOOLFORGE_DOCKER_PORT}
+
+    assets:
+        build:
+            dockerfile: ./.docker/assets/Dockerfile
+            context: .
+        container_name: svgtranslate_assets
```

Comment:
This isn't necessary, compose will give it a name for you. :)

Reply:
Actually, I remembered: we refer to the containers by name in .travis.yml (and in the docs, but that's maybe less important). Any ideas on how to run the tests in a better way?

Reply:
Replace `docker exec` with `docker-compose exec` https://docs.docker.com/compose/reference/exec/ it uses the name of the service rather than the name of the container.